http/swagger.yml

```diff
@@ -3940,6 +3940,8 @@ paths:
       summary: List all organizations
       parameters:
         - $ref: "#/components/parameters/TraceSpan"
+        - $ref: "#/components/parameters/Offset"
+        - $ref: "#/components/parameters/Limit"
```

Comment:
Seems like an entry for `descending` should go in here as well.

Reply:
Done.